### PR TITLE
Enhance/process callback job

### PR DESCRIPTION
**Pull request by Wei-LiangChew**
- adds `ProcessMessengerCallbackJob` to callback generators
- modifies `WebhooksController` to call `ProcessMessengerCallbackJob` instead of the CallbackRegistry
- updated README

### CODE REVIEW

File: lib/messenger_platform/callback_registry.rb [hunk 1]
```diff
@@ -11,14 +11,8 @@ def register(klass)
...
 
-      def process_request(raw)
-        json = JSON.parse(raw)
-        CallbackParser.new(json.deep_dup).parse do |event|
-          @callbacks[event.webhook_name].constantize.new.run(event, json.deep_dup) if @callbacks[event.webhook_name]
-        end
-        true
-      rescue JSON::ParserError => e
-        false
+      def get_callback_handler_for(webhook_name)
```
Comment (Drenmi) [inline lib/messenger_platform/callback_registry.rb]: Not sure the `get_` prefix adds any value here.

Comment (Drenmi) [inline lib/messenger_platform/callback_registry.rb]: Since this lives in the `CallbackRegistry`, even `callback` could arguably be redundant.

Comment (Wei-LiangChew) [inline lib/messenger_platform/callback_registry.rb]: Point taken

**Comment by Wei-LiangChew** [comment]
Hmm, there was a failing spec was due to it not being able to resolve `ProcessMessengerCallbackJob`, which is because it is a generated file... I don't like it, but I resolved the failing spec by copying the generation template into `app/jobs`. I'm thinking it would be better if it was in the `MessengerPlatform` namespace (then I would shorten the name to `ProcessCallback` also)

**Pull request merged**